content/post/know-your-aws-community-day-india-2020-speaker-mayank-raj.md

```diff
@@ -0,0 +1,19 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T15:36:47Z
+description = "Mayank Raj will be speaking on \"Serverless Cron with Second level Schedule and Conditional flow\" at <time> on <date> at AWS Community Day India 2020."
```

Comment:
placeholders should be updated/removed

Reply:
Corrected.

Reply:
@dA505819 the description hasn't been updated

Reply:
@SathyaBhat Corrected.